WebContent/topnav.html

```diff
@@ -133,6 +129,10 @@ <h5 class="modal-title translate" data-i18n="Reset Password" id="exampleModalLab
     </div>
   </div>
 </div>
-
+<script src="resources/lib/url/url.min.js"></script>  
+<script src="resources/i18next.min.js" ></script>
+<script src="resources/jquery-i18next.min.js" ></script>
+<script src="resources/i18nextXHRBackend.js"></script>
+<script src="resources/global.js"></script>
 
```

Comment:
We could either move all the script references to the bottom or to the top (whatever the proper convention is).